gpos/xml/scap/ssg/content/ssg-chainguard-xccdf/OvalDefinitions/DetectOpenSslTest.xml

```diff
@@ -59,5 +67,15 @@
       <independent:pattern operation="pattern match">^P:openssl-provider-fips(-\d+\.\d+\.\d+)?$</independent:pattern>
       <independent:instance datatype="int">1</independent:instance>
     </independent:textfilecontent54_object>
+    <independent:textfilecontent54_object xmlns:independent="http://oval.mitre.org/XMLSchema/oval-definitions-5#independent" id="oval:org.OpenSsl:obj:5" version="1">
+      <independent:filepath>/etc/ssl/openssl.cnf</independent:filepath>
+      <independent:pattern operation="pattern match">^\s*\.include\s+fipsmodule\.cnf\s*$</independent:pattern>
+      <independent:instance datatype="int" operation="greater than or equal">1</independent:instance>
+    </independent:textfilecontent54_object>
+    <independent:textfilecontent54_object xmlns:independent="http://oval.mitre.org/XMLSchema/oval-definitions-5#independent" id="oval:org.OpenSsl:obj:6" version="1">
+      <independent:filepath>/etc/ssl/openssl.cnf</independent:filepath>
+      <independent:pattern operation="pattern match">^\s*fips\s*=\s*fips_sect\s*$</independent:pattern>
```

Comment:
This is looking for:
```ini
[provider_sect]
fips = fips_sect
```
right?

Should we also check for `activate = 1` within that section in `/etc/ssl/fipsmodule.cnf`?